### PR TITLE
update deploy IoT Edge device doc

### PR DESCRIPTION
**Pull request by linyguo**

### CODE REVIEW

**Review by lirenjie95 — APPROVED** [review]

File: docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md [hunk 1]
```diff
@@ -73,20 +76,84 @@ You need to prepare a Linux virtual machine or physical device for IoT Edge. In
...
+
 ## OPTION 1: Use Maestro
 
 To use this option, first install Maestro and the Symphony API. For more information, see [Use Symphony with the Maestro CLI tool](./quick_start_maestro.md).
 
 Once you have maestro installed, you can launch this sample with the following command:
 
 ```bash
-maestro samples run hello-iot-edge --set iot-hub-key=<REPLACE_WITH_HUB_KEY> --set iot-hub-name=<REPLACE_WITH_HUB_NAME> --set device-name=s8c-vm
+maestro samples run hello-iot-edge --set iot-hub-key=<REPLACE_WITH_HUB_KEY> --set iot-hub-name=<REPLACE_WITH_HOST_NAME> --set device-name=s8c-vm
```
Comment (Haishi2016) [inline docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md:136]: IoT Hub name and IoT Hub Host name are two different things. The IoT Hub name is the name of the hub, and the host name is the fully qualified hub name. Please double check the right term is used and hereafter.

Comment (linyguo) [reply docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md:136]: Yes, I have tested locally with the host name. It seems that in the upgraded iot-edge package version, the host name is used in the URL retrieving module twin.

File: docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md [hunk 2]
```diff
@@ -73,20 +76,84 @@ You need to prepare a Linux virtual machine or physical device for IoT Edge. In
...
+# check the latest aziot-identity-service version
+wget -qO- https://raw.githubusercontent.com/Azure/azure-iotedge/main/product-versions.json | jq -r '
+  .channels[]
+  | select(.name == "stable").products[]
+  | select(.id == "aziot-edge").components[]
+  | select(.name == "aziot-identity-service").version
+'
+
+# download and install 
+wget https://github.com/Azure/azure-iotedge/releases/download/<AZIOT_EDGE_VERSION>/aziot-identity-service_<AZIOT_IDENTITY_SERVICE_VERSION>-1_ubuntu22.04_amd64.deb -O aziot-identity-service.deb
```
Comment (RemindD) [inline docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md:105]: 1. Add definition for AZIOT_EDGE_VERSION and AZIOT_IDENTITY_SERVICE_VERSION
2. The first parameter should be AZIOT_IDENTITY_SERVICE_VERSION or AZIOT_EDGE_VERSION? 
3. maybe switch the order of aziot-identity-service.deb and aziot-edge.deb to keep consistency with the previous section.

File: docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md [hunk 3]
```diff
@@ -55,6 +55,9 @@ You need to prepare a Linux virtual machine or physical device for IoT Edge. In
 
   # SSH into the machine
   ssh hbai@<public IP of your VM>
```
Comment (Haishi2016) [inline docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md:57]: Remove reference to "hbai"

File: docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md (hunk 2 above)
Comment (linyguo) [reply docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md]: Added definitions and updated the sequence. The first parameter is `AZIOT_EDGE_VERSION`, they are under a same parent version.

**Review by msftcoderdjw — APPROVED** [review]

**Review by Haishi2016 — APPROVED** [review]

**Pull request merged**